lib/plug/cowboy/translator.ex

```diff
@@ -84,6 +92,14 @@ defmodule Plug.Cowboy.Translator do
     ["Request: ", method, ?\s, path_to_iodata(conn.request_path, query_string), ?\n]
   end
 
+  defp maybe_add_conn_metadata(metadata, conn) do
+    if Application.get_env(:plug_cowboy, :conn_in_exception_metadata, true) do
+      Keyword.put(metadata, :conn, conn)
+    else
+      metadata
+    end
+  end
```

Comment:
With the suggestion above this becomes:

```suggestion
  defp maybe_conn_metadata(metadata, conn) do
    if Application.get_env(:plug_cowboy, :conn_in_exception_metadata, true) do
      [conn: conn]
    else
      []
    end
  end
```